pkg/server/server.go

```diff
@@ -476,3 +646,264 @@ func adaptContext(parent genericapiserver.PostStartHookContext) context.Context
 	}(parent.StopCh)
 	return ctx
 }
+
+type kcpCRDLister struct {
```

Comment:
I should probably comment why this exists/what it does.

Reply:
yes, godoc please :)